lib/rules/require-valid-alt-text.js

```diff
@@ -64,6 +64,14 @@ module.exports = class RequireValidAltText extends Rule {
             }
           }
 
+          // src value is dynamic and alt value is not
+          if (srcValue && srcValue.type && altValue && !altValue.type) {
+            this.logNode({
+              message: 'The `alt` attribute value must be of the same type as `src` attribute',
```

Comment:
this may be little bit wrong for case like:

```hbs
{{each this.levels as |level index|}}
  <img src={{level.src}} alt={{concat "Photo of car with index" index}} />
{{/each}}
```

Reply:
Ya, good call out here.